### PR TITLE
Movesegm

### PR DESCRIPTION
**Pull request by patateqbool**
This change adds support for the mov segment feature

### CODE REVIEW

File: idarling/core/events.py [hunk 1]
```diff
@@ -336,6 +336,11 @@ def __call__(self):
                 ti = get_idati()
                 ordinal = ctypes.c_uint32(py_ord)
                 ntf_flags = ctypes.c_int(ida_typeinf.NTF_REPLACE)
+                name = Event.encode_bytes(name)
```
[the remark this answers is not in the document]
Comment (patateqbool) [reply idarling/core/events.py]: Can we merge even if I didn't branch from master ?

[the remark this answers is not in the document]
Comment (NeatMonster) [reply idarling/core/events.py]: Not as is, you can see [here](https://github.com/IDArlingTeam/IDArling/pull/75/files) that it has the old changes from your other branch.

[the remark this answers is not in the document]
Comment (patateqbool) [reply idarling/core/events.py]: I've tried a fix, I don't know if it's the good way to do it...

**Comment by patateqbool** [comment]
I've tried a fix, I don't know if it's the good way to do it...

**Pull request merged**